### ISSUE TITLE
Is version 2 ready for use?

### ISSUE THREAD

**Issue by kidmeke** [alexandersimoes/d3plus#619]
Hi, I don't have an issue as such, but I'm contacting you through the github issues page as suggested on https://d3plus.org/examples/.

I am (**_was_**) confused about the status of the project. I see that you have documentation for version 2 but I can only find a download for version 1.9.8. Actually, I just checked again and I see that downloadable version is now 2.0! Must have happened sometime in the past week or two - I've been meaning to ask this question for a while and am only now getting around to it.

So my questions changes slightly. I see that the version is 2.0.0-alpha.15, yet it is the only version available for download. Is this version ready for prime-time or should I continue to use version 1.9.8?

My confusion was that the main website referred to version 2 which didn't seem to be available and I couldn't find any information re: the status/timeline etc. The last blog post is from 2015, so I was left wondering if the project was abandoned and if not, where should I be looking to keep abreast of d3plus developments.

TL;DR:

- Should I use version 2?
- Where should I be looking to keep informed about recent activities with the project (such as version 2 being released)?

**Comment by davelandry** [comment]
@kidmeke @lawlesst apologies for the radio silence. We've been transitioning development from version 1.0 over to 2.0 over the past year, and we're nearing release. Here's a write-up I just added to the master branch README:

> ⚠️  **Version 2 Incoming**  ⚠️
> 
> This branch is for version 1, which is not being actively maintained. All development has been focused on version 2, which splits the project into [many small libraries](https://github.com/d3plus/) that are designed to work together.
> 
> There will be an announcement when the version 2 modules are feature-complete with everything from version 1, but the water is warm so don't be afraid to jump in.
>
>**[Click here to view the D3plus 2.0 branch](https://github.com/alexandersimoes/d3plus/tree/2.0)**

At my company, [Datawheel](http://www.datawheel.us/), we've been using the new 2.0 modules exclusively over the past year, and have even launched some sites using them:
* [Data Africa](https://dataafrica.io/)
* [DataChile](https://es.datachile.io/)

Any help in testing these new modules would be greatly appreciated. We are in the process of creating the final punchlist of features that need to be ported over to 2.0, and will be updating the README of the 2.0 branch with the progress.
